admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -466,6 +467,84 @@ In migrated Nextcloud installations the old behavior still applies, which is usi
 
   sudo -u www-data php occ config:app:set user_ldap enforce_home_folder_naming_rule --value=1
 
+"$home" Placeholder Field:
+  $home in an external storage configuration will be replaced with the value of the specified attribute
+
```

Comment:
This change is not mine!
The $home is still there, but those three lines in the documentation were removed somewhen between April 2022 and February 2023.

```suggestion
```

Should I kick this from my pull request? I'm really unsure about this.

Reply:
I removed it.